pkg/config/config.go

```diff
@@ -0,0 +1,109 @@
+package config
+
+import (
+	"strconv"
+
+	"github.com/pelletier/go-toml"
+)
+
+// DefaultConfig for the DC/OS CLI.
+var DefaultConfig = Config{
+	Core: Core{
+		Timeout: 180,
+	},
+}
+
+// Config for the DC/OS CLI. The "core" and "cluster" sections are mapped to structs.
+// Additional sections such as "marathon" are stored in a Tree.
+// This difference is due to the fact that we know the content of "core" and "cluster"
+// but not the content of additional sections.
+type Config struct {
+	Core    Core    `toml:"core"`
+	Cluster Cluster `toml:"cluster"`
+	tree    *toml.Tree
+}
+
+// Core is the main section of the DC/OS CLI configuration. It contains the DC/OS URL, an authentication
```

Comment:
When cutting a comment, let's make the length similar for each line or cut using periods. E.g. here:
```
// Core is the main section of the DC/OS CLI configuration. 
// It contains  the DC/OS URL, an authentication token and various other config options.
```